articles/integrations/using-templates-sparkpost-wordpress.md

```diff
@@ -30,6 +30,8 @@ To use these variables in your templates, you'll have to use our [Templates API
 }
 ```
 
-Note that content is enclosed in triple curly braces - this allows HTML content to be substituted. In the WordPress admin, on the settings screen for the SparkPost plugin, remember to fill in the Template field (in this example wordpress-testing):
+Note that `content` is enclosed in triple curly braces - this allows HTML content to be substituted without escaping. In the WordPress admin, on the settings screen for the SparkPost plugin, remember to fill in the Template field (in this example wordpress-testing):
 
 ![](media/using-templates-sparkpost-wordpress/wordpress_templates_screenshot_original.png)
+
+You can modify template name and/or substitution tags dynamically using [hooks supported by SparkPost plugin](https://github.com/SparkPost/wordpress-sparkpost/blob/master/docs/hooks.md). 
```

Comment:
This line could use either 
- a "the" in front of "template name": "You can modify the template name and/or"
- an "s" in "name": "You can modify template names and/or"